app/controllers/CreateGroupSelectTeamMembersController.scala

```diff
@@ -48,6 +48,7 @@ class CreateGroupSelectTeamMembersController @Inject()
   mcc: MessagesControllerComponents,
   val sessionCacheService: SessionCacheService,
   group_created: group_created,
+  TaxGroup_created: TaxGroup_created,
```

Comment:
TaxGroup_created: TaxGroup_created, keep the snake case style for view templates

eg. tax_group_created